### PR TITLE
Add Test Adapter

### PR DESCRIPTION
**Pull request by gmartin7**
* Needed to run tests in Visual Studio, if no Resharper

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1164)

### CODE REVIEW

File: TestApps/ClipboardTestApp/ClipboardTestApp.csproj [hunk 1]
```diff
@@ -5,6 +5,10 @@
 		<UseWindowsForms>true</UseWindowsForms>
 	</PropertyGroup>
 
+	<ItemGroup>
+	  <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
+	</ItemGroup>
```
Comment (andrew-polk) [inline TestApps/ClipboardTestApp/ClipboardTestApp.csproj:10]: I don't think this project needs it.

File: TestApps/FastSplitterTest/FastSplitterTest.csproj [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="GitVersion.MsBuild" Version="5.6.8" PrivateAssets="all" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
```
Comment (andrew-polk) [inline TestApps/FastSplitterTest/FastSplitterTest.csproj:14]: I don't think this project needs it. (Notice there is no Nunit package reference)

File: TestApps/ReportingTest/Reporting.TestApp.csproj [hunk 3]
```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="GitVersion.MsBuild" Version="5.6.8" PrivateAssets="all" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
```
Comment (andrew-polk) [inline TestApps/ReportingTest/Reporting.TestApp.csproj:14]: I don't think this project needs it. (Notice there is no Nunit package reference)

File: TestApps/SIL.Email.TestApp/SIL.Email.TestApp.csproj [hunk 4]
```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="GitVersion.MsBuild" Version="5.6.8" PrivateAssets="all" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
```
Comment (andrew-polk) [inline TestApps/SIL.Email.TestApp/SIL.Email.TestApp.csproj:14]: I don't think this project needs it. (Notice there is no Nunit package reference)

File: TestApps/SIL.Windows.Forms.TestApp/SIL.Windows.Forms.TestApp.csproj [hunk 5]
```diff
@@ -21,6 +21,7 @@
     <PackageReference Include="GitVersion.MsBuild" Version="5.6.8" PrivateAssets="all" />
     <PackageReference Include="Icu4c.Win.Min" Version="56.1.82" />
     <PackageReference Include="L10NSharp" Version="5.0.0-beta0059" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
```
Comment (andrew-polk) [inline TestApps/SIL.Windows.Forms.TestApp/SIL.Windows.Forms.TestApp.csproj:24]: I don't think this project needs it. (Notice there is no Nunit package reference)

File: TestApps/TestAppKeyboard/TestAppKeyboard.csproj [hunk 6]
```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <PackageReference Include="GitVersion.MsBuild" Version="5.6.8" PrivateAssets="all" />
     <PackageReference Include="NDesk.DBus" Version="0.15.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
```
Comment (andrew-polk) [inline TestApps/TestAppKeyboard/TestAppKeyboard.csproj:15]: I don't think this project needs it. (Notice there is no Nunit package reference)

**Review by andrew-polk — CHANGES_REQUESTED** [review]

**Review by gmartin7 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1164)* status: 0 of 24 files reviewed, 6 unresolved discussions (waiting on @andrew-polk)

___
*[`TestApps/FastSplitterTest/FastSplitterTest.csproj` line 14 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/1164#-N1np3RSZCzpxqa28I1r-r1-14:-N1nrysr3qaTAIFAYM6R:b-896fix) ([raw file](https://github.com/sillsdev/libpalaso/blob/667c64c916ba53537b3f83c07e455da7aeb06116/TestApps/FastSplitterTest/FastSplitterTest.csproj#L14)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

I don't think this project needs it. (Notice there is no Nunit package reference)
</blockquote></details>

Done.
___
*[`TestApps/ReportingTest/Reporting.TestApp.csproj` line 14 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/1164#-N1np3RSZCzpxqa28I1s-r1-14:-N1ns8o24FYa8rP_oEhp:b-896fix) ([raw file](https://github.com/sillsdev/libpalaso/blob/667c64c916ba53537b3f83c07e455da7aeb06116/TestApps/ReportingTest/Reporting.TestApp.csproj#L14)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

I don't think this project needs it. (Notice there is no Nunit package reference)
</blockquote></details>

Done.
___
*[`TestApps/SIL.Email.TestApp/SIL.Email.TestApp.csproj` line 14 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/1164#-N1np3RSZCzpxqa28I1t-r1-14:-N1nsGm313fgRHJ6KBs1:b-896fix) ([raw file](https://github.com/sillsdev/libpalaso/blob/667c64c916ba53537b3f83c07e455da7aeb06116/TestApps/SIL.Email.TestApp/SIL.Email.TestApp.csproj#L14)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

I don't think this project needs it. (Notice there is no Nunit package reference)
</blockquote></details>

Done.
___
*[`TestApps/TestAppKeyboard/TestAppKeyboard.csproj` line 15 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/1164#-N1np3RSZCzpxqa28I1v-r1-15:-N1nslzp54--whCNbki-:b-896fix) ([raw file](https://github.com/sillsdev/libpalaso/blob/667c64c916ba53537b3f83c07e455da7aeb06116/TestApps/TestAppKeyboard/TestAppKeyboard.csproj#L15)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

I don't think this project needs it. (Notice there is no Nunit package reference)
</blockquote></details>

Done.
___
*[`TestApps/ClipboardTestApp/ClipboardTestApp.csproj` line 10 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/1164#-N1np3RSZCzpxqa28I1q-r1-10:-N1nrcxJ4fUjS63wcJaD:bj5ih0f) ([raw file](https://github.com/sillsdev/libpalaso/blob/667c64c916ba53537b3f83c07e455da7aeb06116/TestApps/ClipboardTestApp/ClipboardTestApp.csproj#L10)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

I don't think this project needs it.
</blockquote></details>

Done. Silly me. I just added it anywhere it mentioned "Test"! A bit of overkill, I guess.
___
*[`TestApps/SIL.Windows.Forms.TestApp/SIL.Windows.Forms.TestApp.csproj` line 24 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/1164#-N1np3RSZCzpxqa28I1u-r1-24:-N1nsbUyAIbGmmbVeZzL:b-896fix) ([raw file](https://github.com/sillsdev/libpalaso/blob/667c64c916ba53537b3f83c07e455da7aeb06116/TestApps/SIL.Windows.Forms.TestApp/SIL.Windows.Forms.TestApp.csproj#L24)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

I don't think this project needs it. (Notice there is no Nunit package reference)
</blockquote></details>

Done.

**Review by andrew-polk — APPROVED** [review]
> Reviewed 18 of 24 files at r1, 6 of 6 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1164)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @gmartin7)

**Pull request merged**